README.md

```diff
@@ -9,16 +9,9 @@ The MoveIt! Motion Planning Framework **for ROS 2.0**
 - [Get Involved](http://moveit.ros.org/documentation/contributing/)
 
 ## Milestones
-- [ ] Setup instructions
-  - [ ] Install instructions
-    - [ ] Ubuntu 18.04
-    - [ ] OS X 10.14
-- [ ] Update/setup infrastructure for development
-  - [x] Delete metapackages
-  - [x] Upgrade continuous integration for ROS 2.0
-  - [x] Refactor/cleanup folder hierarchy
+- [ ] Install instructions for Ubuntu 18.04 in README.md
+- [ ] Upgrade continuous integration for ROS 2.0
 - [ ] Convert moveit packages to ROS 2.0
-  - [x] ~~Include moveit_msgs in HRIM (**@ibaiape**)~~
```

Comment:
@vmayoral what is HRIM?

Reply:
@davetcoleman https://github.com/AcutronicRobotics/HRIM, discarded now as part of our contributions due to community's input. Refer to https://github.com/ros2/ros2/issues/607#issuecomment-462938420